tests/ops/op_math/test_controlled_ops.py

```diff
@@ -88,7 +140,7 @@ def test_wrong_shape(self):
         with pytest.raises(ValueError, match=r"Input unitary must be of shape \(2, 2\)"):
             qml.ControlledQubitUnitary(np.eye(4), control_wires=[0, 1], wires=2).matrix()
 
-    @pytest.mark.parametrize("target_wire", range(3))
+    @pytest.mark.parametrize("target_wire", list(range(3)))
```

Comment:
Does this imply that we don't accept iterable as wires input anymore?

Reply:
No, the only thing that has changed is the test parametrization syntax, but the target wire before and after the change will stay the same (int scalar).